src/pages/BoardPage/Notice/NoticeDetailPage/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useParams } from 'react-router-dom';
+import PostDetailHeader from 'components/Post/PostDetail/PostDetailHeader';
+import PostDetailContent from 'components/Post/PostDetail/PostDetailContent';
+import useArticleDetail from 'pages/BoardPage/Notice/hooks/useArticleDetail';
+
+function NoticeDetailPage() {
+  const params = useParams();
+  const articleDetail = useArticleDetail(params.id);
+
+  return (
+    <>
+      <PostDetailHeader
+        boardId={articleDetail!.board_id}
+        title={articleDetail!.title}
+        createdAt={articleDetail!.created_at}
+        commentCount={articleDetail!.comment_count}
+        nickname={articleDetail!.nickname}
+        hit={articleDetail!.hit}
+      />
+      <PostDetailContent
+        content={articleDetail!.content}
+      />
```

Comment:
타입 단언보다는 해당 데이터가 있을 때 렌더링 하는건 어떨까요? 전 그렇게 했었어요.
```suggestion
    <div>
      {articleDetail && (
      <>
        <PostDetailHeader
          boardId={articleDetail.board_id}
          title={articleDetail.title}
          createdAt={articleDetail.created_at}
          commentCount={articleDetail.comment_count}
          nickname={articleDetail.nickname}
          hit={articleDetail.hit}
        />
        <PostDetailContent
          content={articleDetail.content}
        />
      </>
      )}
    </div>
```

Reply:
[c3ace06](https://github.com/BCSDLab/KOIN_WEB_RECODE/pull/30/commits/c3ace064f57ce9eb95365606eacac21a4b0ec6dd)
제안해주신대로 변경 완료했습니다